src/pages/api/auth/[...nextauth].ts

```diff
@@ -29,47 +29,40 @@ export const authOptions: NextAuthOptions = {
   },
 
   providers: [
-    Credentials({
+    CredentialsProvider({
       name: 'NUS Email',
-      credentials: {
-        email: {
-          label: 'NUS Email',
-          type: 'email',
-          placeholder: 'XXX@u.nus.edu',
-        },
-        password: {
-          label: 'Password',
-          type: 'password',
-        },
-      },
+      credentials: {},
```

Comment:
Hi Zhermin, out of curiosity, did you remove this portion because it does not integrate well with your part?